src/contexts/SpaceDataProvider.js

```diff
@@ -24,7 +25,7 @@ const SpaceDataProvider = ({ children }) => {
 
   useEffect(() => {
     const fetchData = async () => {
-      const baseUrl = process.env.REACT_APP_BASE_URL;
+      const baseUrl = REACT_APP_BASE_URL;
```

Comment:
Any reason why you don't pass the constant directly?

Reply:
you're right, no reason